src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -232,8 +233,8 @@ export class MeetingManager implements AudioVideoObserver {
     this.meetingRegion = meetingInfo.MediaRegion;
     this.meetingId = this.configuration.meetingId;
 
-    if (eventReporter) {
-      this.eventReporter = eventReporter;
+    if (eventController) {
+      this.eventController = eventController;
```

Comment:
We want to get rid of storing this as a global variables. I think it's fine for now. 
Later in Kunal's PR we could remove this and use the argument one directly.

Reply:
Gotcha, can't really change it in my PR rn as it's needed to be global until Kunal's changes